map/src/context/LocalTrackGenerator.js

```diff
@@ -0,0 +1,47 @@
+function generate(ctx) {
```

Comment:
Could be combined with LocalTracksStorage to LocalTracksManager? 
Too small files

Reply:
`+`